docs/source/ops/index.rst

```diff
@@ -0,0 +1,86 @@
+
+Still, DAML applications may be affected in the following ways:
+
+- Pruning is potentially a long-running operation and demanding one in terms of system resources; as such, it may significantly reduce DAML Ledger API throughput and increase latency while it is being performed. It is thus recommended to plan pruning invocations when low system utilization is expected.
+- Pruning may degrade the behavior of and/or crash the system if the pruning offset is too recent. In particular, the system might misbehave if command completions are pruned before the command trackers are able to process the completions.
+- Pruning may affect the behavior of Ledger API calls that allow to read data from the ledger: see the next sub-section for more information about API impacts.
+
+How the DAML Ledger API is affected
+-----------------------------------
+
+- Active data streams from the DAML Participant may abort and need to be re-established by the DAML application from a later offset than pruned, even if they are already streaming past it.
+- Requesting information at offsets that predate pruning, including from the ledger's start, will result in a ``NOT_FOUND`` gRPC error.
+  - As a consequence, after pruning, a DAML application must bootstrap from the Active Contract Service and a recent offset [3]_.
+
+Submission validation and DAML Ledger API endpoints that write to the ledger are generally not affected by pruning; an exception is that in-progress calls could abort while awaiting completion.
+
+Please refer to the :doc:`protobuf documentation of the API </app-dev/grpc/proto-docs>` for details about the ``prune`` operation itself and the behavior of other DAML Ledger API endpoints when pruning is being or has been performed.
+
+Other limitations
+-----------------
+
+- If the number of divulged contracts continually grows, the storage used by the participant node may still keep growing, even with regular pruning.
+- Pruning may be rejected even if the node is running correctly (for example, to preserve non-repudiation properties); in this case, the application might not be able to archive contracts containing PII or pruning of these contracts may not be possible; thus, actually deleting this PII may also be technically unfeasible.
+- Pruning may leave parties, packages, and configuration data on the participant node, even if they are no longer needed for transaction processing, and even if they contain PII [3]_.
+- Pruning does not move pruned information to cold storage but simply deletes pruned data; for this reason, it is advisable to back up the Participant Index DB before invoking pruning. See the next sub-section for more Participant Index DB-related advice before and after invoking `prune`.
+- Pruning is not selective but rather effectively truncates the ledger, removing the transactions and archived contracts at and before the pruning offset.
```

Comment:
- Pruning deletes data from the underlying database and therefore frees up space *within* the database, which can and will be reused during the continued operation of the Index DB. Whether this freed up space is handed back to the OS depends on the database in use. For Postgres for example, the deleted data frees up space in the table storage itself, but does not shrink the size of the files backing the tables of the IndexDB. See the documentation on [`VACUUM` and `VACUUM FULL`](https://www.postgresql.org/docs/12/sql-vacuum.html) for more information..

Reply:
@fabiotudone-da : this comment can be ignored or worked into the comments below. I've written it before seeing that.

Reply:
I think it is valuable, I'll work it out into the content on the same subject.

Reply:
Integrated.